completions/sudo

```diff
@@ -23,7 +23,7 @@ _sudo()
                 break
             fi
             # shellcheck disable=SC2254
-            [[ ${words[i]} == -@(user|other-user|group|close-from|prompt|${noargopts}[uUgCp]) ]] &&
+            [[ ${words[i]} == @(--@(user|other-user|group|close-from|prompt)|-${noargopts}[uUgCp]) ]] &&
```

Comment:
Cherry picked to master as 1cf6fa6aadcd6162cdbe78e9462f611bfcd0fdb0.

Reply:
Thanks